### PR TITLE
Feature/embed notebook

### PR DESCRIPTION
**Pull request by DerpsBurps**
Embeds notebooks into posts. There are a few problems, such as the embed being forced into a certain windows size rather than dynamically resizing, and the embed overlapping parts of the footer.

### CODE REVIEW

File: computingblog.py [hunk 1]
```diff
@@ -71,4 +81,4 @@ def get_banner_image(filename):
     if len(sys.argv) > 1 and sys.argv[1] == "build":
         freezer.freeze()
     else:
-        app.run(ssl_context='adhoc')
```
Comment (DerpsBurps) [inline computingblog.py]: I didn't put this back, because I somewhat forgot.

Comment (nshdesai) [reply computingblog.py]: No worries. I see you have fixed it. Awesome!

**Comment by nshdesai** [comment]
Great Code. Off to the press!

**Comment by nshdesai** [comment]
I think the code has some integration issues

```
2018-11-14T23:28:45.931403+00:00 app[web.1]: [2018-11-14 23:28:45 +0000] [12] [ERROR] Socket error processing request.
2018-11-14T23:28:45.931417+00:00 app[web.1]: Traceback (most recent call last):
2018-11-14T23:28:45.931420+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/sync.py", line 135, in handle
2018-11-14T23:28:45.931422+00:00 app[web.1]: self.handle_request(listener, req, client, addr)
2018-11-14T23:28:45.931424+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
2018-11-14T23:28:45.931425+00:00 app[web.1]: six.reraise(*sys.exc_info())
2018-11-14T23:28:45.931427+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/six.py", line 625, in reraise
2018-11-14T23:28:45.931430+00:00 app[web.1]: raise value
2018-11-14T23:28:45.931431+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/sync.py", line 176, in handle_request
2018-11-14T23:28:45.931433+00:00 app[web.1]: respiter = self.wsgi(environ, resp.start_response)
2018-11-14T23:28:45.931435+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/app.py", line 2309, in __call__
2018-11-14T23:28:45.931436+00:00 app[web.1]: return self.wsgi_app(environ, start_response)
2018-11-14T23:28:45.931438+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/app.py", line 2295, in wsgi_app
2018-11-14T23:28:45.931440+00:00 app[web.1]: response = self.handle_exception(e)
2018-11-14T23:28:45.931441+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/app.py", line 1741, in handle_exception
2018-11-14T23:28:45.931443+00:00 app[web.1]: reraise(exc_type, exc_value, tb)
2018-11-14T23:28:45.931444+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
2018-11-14T23:28:45.931446+00:00 app[web.1]: raise value
2018-11-14T23:28:45.931448+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
2018-11-14T23:28:45.931449+00:00 app[web.1]: response = self.full_dispatch_request()
2018-11-14T23:28:45.931451+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
2018-11-14T23:28:45.931452+00:00 app[web.1]: rv = self.handle_user_exception(e)
2018-11-14T23:28:45.931455+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
2018-11-14T23:28:45.931456+00:00 app[web.1]: reraise(exc_type, exc_value, tb)
2018-11-14T23:28:45.931458+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
2018-11-14T23:28:45.931459+00:00 app[web.1]: raise value
2018-11-14T23:28:45.931461+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
2018-11-14T23:28:45.931463+00:00 app[web.1]: rv = self.dispatch_request()
2018-11-14T23:28:45.931464+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
2018-11-14T23:28:45.931466+00:00 app[web.1]: return self.view_functions[rule.endpoint](**req.view_args)
2018-11-14T23:28:45.931468+00:00 app[web.1]: File "/app/computingblog.py", line 49, in notebook
2018-11-14T23:28:45.931469+00:00 app[web.1]: return render_template(notebook)
2018-11-14T23:28:45.931471+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/templating.py", line 134, in render_template
2018-11-14T23:28:45.931473+00:00 app[web.1]: return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
2018-11-14T23:28:45.931475+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/jinja2/environment.py", line 869, in get_or_select_template
2018-11-14T23:28:45.931476+00:00 app[web.1]: return self.get_template(template_name_or_list, parent, globals)
2018-11-14T23:28:45.931478+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/jinja2/environment.py", line 830, in get_template
2018-11-14T23:28:45.931479+00:00 app[web.1]: return self._load_template(name, self.make_globals(globals))
2018-11-14T23:28:45.931481+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/jinja2/environment.py", line 804, in _load_template
2018-11-14T23:28:45.931483+00:00 app[web.1]: template = self.loader.load(self, name, globals)
2018-11-14T23:28:45.931485+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/jinja2/loaders.py", line 113, in load
2018-11-14T23:28:45.931486+00:00 app[web.1]: source, filename, uptodate = self.get_source(environment, name)
2018-11-14T23:28:45.931488+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/templating.py", line 58, in get_source
2018-11-14T23:28:45.931489+00:00 app[web.1]: return self._get_source_fast(environment, template)
2018-11-14T23:28:45.931491+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.6/site-packages/flask/templating.py", line 86, in _get_source_fast
2018-11-14T23:28:45.931493+00:00 app[web.1]: raise TemplateNotFound(template)
2018-11-14T23:28:45.931555+00:00 app[web.1]: jinja2.exceptions.TemplateNotFound: custom.css
```
Here is the stack trace of the error

**Pull request merged**